pleroma/emoji-index/QueerCats/index.json

```diff
@@ -0,0 +1,10 @@
+{
+    "queerCats": {
+      "src_sha256": "737A7C1DEE4599043C88EF39885A4FEFBF27F1CD5C23FC00609E6E42EB4E8C10",
+      "src": "https://raw.githubusercontent.com/mcrosson/QueerCats/pleroma-emoji-pack/PNG/PNG.zip",
+      "license": "CC0",
+      "homepage": "https://github.com/ZoeBijl/QueerCats",
+      "files": "https://raw.githubusercontent.com/mcrosson/QueerCats/pleroma-emoji-pack/pleroma/emoji-index/QueerCats/queerCats.json",
```

Comment:
Same as for the readme

Reply:
Correct, those were placeholders I used for testing and I didn't want to break the links when I submitted the PR in case others were testing.